applications/GeoMechanicsApplication/tests/prescribed_derivative_tests/README.md

```diff
@@ -0,0 +1,40 @@
+# Prescribed derivatives
+
+The tests in this folder assert that prescribed derivatives are applied correctly to the models and are not overwritten.
+The
+three derivatives that are prescribed in these tests are acceleration, velocity and dt_water_pressure (the time
+derivative of the water
+pressure). After running the simulation, it is checked that on the nodes to which the prescribed derivatives are
+applied, the actual derivatives match the prescribed ones.
+
+## Setup
+
+The setup for the test case is a 10-element 2D mesh with a 2D linear elastic material. The schematic depiction of the
+mesh is shown below:
+
+![MeshStructure](MeshStructure.svg)
+
+The derivatives (acceleration, velocity and dt_water_pressure) are prescribed to node 1, 2 and 3, using the following
+table:
+
+| TIME | VALUE  |
+|------|--------|
+| 0.05 | -0.01  |
+| 0.1  | -0.005 |
+| 0.15 | -0.003 |
+| 0.2  | 0.01   |
+
+- Constraints:
+    - The Y displacements at the sides and bottom are fixed.
+- Material:
+    - The material is described using a linear elastic material with a GeoLinearElasticPlaneStrain2DLaw, a Young's
+      modulus
+      of 150000 kPa and a Poisson ratio of 0.3.
+
+No conditions or loads are applied.
```

Comment:
Not true, the bottom has a Dirichlet condition for a, v or dp/dt. That is a form of loading.

Reply:
Good point, added